reviewer-guide.en.md

```diff
@@ -44,10 +45,29 @@ There are four different vote options: -1, -0, +0, +1.
 
 Each vote option's meaning is as follows:
 * **-1**：Strong Reject    This **SHOULD NOT** be in PyCon TW 2025
-* **-0**：Weak Reject
-* **+0**：Weak Accept
+* **-0**：Weak Reject    This **IS ACCEPTABLE, BUT SLIGHTLY NOT RECOMMENDED** to be in PyCon TW 2025 
+* **+0**：Weak Accept    This **IS ACCEPTABLE, AND SLIGHTLY RECOMMENDED** to be in PyCon TW 2025
 * **+1**：Strong Accept    This **SHOULD** be in PyCon TW 2025
 
+# Reviewer's Responsibility and Review Rate
+Dear Reviewers,
+Thank you very much for agreeing to serve as a reviewer for this conference. Your professional contributions are essential to ensuring the technical quality of the conference. We understand that you have a busy schedule and other personal commitments. However, your participation plays a critical role in enhancing the quality and fairness of the conference.
+
+In order to ensure a smooth review process and to guarantee that each proposal is reviewed in a timely and thorough manner, we hope to achieve the following common goals:
+
+### 1. Expectations for Review Rate
+We encourage each reviewer to take responsibility for reviewing proposals in fields they are familiar with. The ideal review rate should reach at least 80% for the proposals within your area of expertise. This will ensure that most proposals receive a sufficient number of reviews, providing appropriate feedback to the authors and ensuring the quality of the conference program.
```

Comment:
Not sure I like the idea of putting the number here. 🤔 might push some reviewers away, But this is non-blocking, as I'm not in a position to decide this. Feel free to ignore this one.